src/main/java/services/RestoreFactory.java

```diff
@@ -414,4 +420,12 @@ public String getAsUsername() {
     public void setAsUsername(String asUsername) {
         this.asUsername = asUsername;
     }
+
+    public boolean isUseLiveQuery() {
```

Comment:
Its Spring standard to have getters and setters for your all your class variables